test/getiBFT_spec.rb

```diff
@@ -11,6 +11,7 @@
   describe "#getiBFT" do
     context "when filtering output of 'iscsiadm -m fw'" do
       it "returns data in form of a map " do
+        Yast::Arch.stub(:architecture).and_return("x86_64")
```

Comment:
I'd even consider enabling the verifying doubles in all tests, like here https://github.com/yast/yast-registration/blob/9cd0dfe8ecea8cc45c236a14f1c3c0a548c1718c/test/spec_helper.rb#L20-L25 .